llvm/test/MC/SystemZ/insn-bad.s

```diff
@@ -4176,12 +4176,9 @@
 #CHECK: may	%f0, %f0, -1
 #CHECK: error: invalid operand
 #CHECK: may	%f0, %f0, 4096
-#CHECK: error: invalid register pair
-#CHECK: may	%f2, %f0, 0
 
 	may	%f0, %f0, -1
 	may	%f0, %f0, 4096
-	may	%f2, %f0, 0
```

Comment:
Oh, one more thing:  in addition to removing the `insn-bad` error check, can you please add a _positive_ test to `insn-good` that this is actually accepted?  This is quite unusual, and it would be good to have a test that catches any accidental regression here.

Reply:
done